katello/katello-restore

```diff
@@ -10,7 +10,10 @@ ONLINE_BACKUP_FILES = ['mongo_dump', 'candlepin.dump', 'foreman.dump', 'pg_globa
 OFFLINE_BACKUP_FILES = ['mongo_data.tar.gz', 'pgsql_data.tar.gz']
 FOREMAN_PROXY_CONTENT_BACKUP_FILES = ['mongo_data.tar.gz']
 FOREMAN_PROXY_CONTENT_ONLINE_BACKUP_FILES = ['mongo_dump']
-FOREMAN_PROXY_CONTENT = "foreman-proxy-content" # keep this variable easy to change for backporting
+
+# keep these variables easy to change for backporting
+FOREMAN_PROXY_CONTENT = "foreman-proxy-content"
+SCENARIO = "katello"
```

Comment:
This will make backporting somewhat easier, but does not solve all problems just a heads up. There is easy potential for cherry picks to overwrite this.